src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfSummaryRequestHandlerTests.cs

```diff
@@ -37,7 +38,7 @@ public GetAatfSummaryRequestHandlerTests()
             mapper = A.Fake<IMapper>();
             evidenceStoredProcedures = A.Fake<IEvidenceStoredProcedures>();
 
-            request = new GetAatfSummaryRequest(fixture.Create<Guid>());
+            request = new GetAatfSummaryRequest(fixture.Create<Guid>(), SystemTime.Now.Year);
```

Comment:
pass fixture.Create<int>() to GetAatfSummaryRequest rather than SystemTime.Now.Year. It allows us to check that the Request Compliance year is used in the tests rather than system time.